roles/common/tasks/python.yml

```diff
@@ -79,7 +79,7 @@
     virtualenv: "{{ common.python.base_venv }}"
     extra_args: "{{ pip_extra_args }}"
   with_items:
-    - shade>=1.9.0
+    - 'shade>=1.9.0,<1.21.0'
```

Comment:
should move this to a variable in common/defaults/main.yml so that in future we can adjust via envs.

Also, do we know what is causing problem wit shade 1.21.0? Maybe we need to pin some client in baseenv.